src/views/Nft/market/Collection/IndividualNFTPage/ForSaleTableCard/ForSaleTableRows.tsx

```diff
@@ -76,14 +77,11 @@ const Row: React.FC<RowProps> = ({ t, nft, bnbBusdPrice, account }) => {
       <Box pl="24px">
         <Flex justifySelf="flex-start" alignItems="center" width="max-content">
           <BinanceIcon width="24px" height="24px" mr="8px" />
-          <Text bold>{nft.marketData.currentAskPrice}</Text>
+          <Text bold>{formatNumber(parseFloat(nft.marketData.currentAskPrice), 0, 6)}</Text>
```

Comment:
Isn't 6 a bit too many? Maybe 3?

Reply:
Done. I think the more the better but 3 or 4 or used in other places.